docs/Guides/Detecting-When-Clients-Abort.md

```diff
@@ -32,12 +32,6 @@ logging purposes or halting business logic.
 
 ### Hands-on
 
-For this sample solution, we'll be using the following:
```

Comment:
This was following the pattern of https://github.com/fastify/fastify/blob/main/docs/Guides/Delay-Accepting-Requests.md, and attempting to avoid confusion if it no longer works in future versions. But I don't mind deleting if those are not concerns.

Reply:
I personally think this adds to the confusion as all the APIs shown here should be fairly stable across major versions of all those packages